app/Http/Controllers/UserController.php

```diff
@@ -216,10 +217,7 @@ public function forgotPassword(Request $request)
     public function resetPassword(ResetPasswordRequest $request)
     {
         $device = Auth::guard('api')->user();
```

Comment:
with middleware previously defined in route (or in constructor), its not necessary to define guard inline